deepchecks/vision/checks/distribution/train_test_label_drift.py

```diff
@@ -49,10 +84,10 @@ class TrainTestLabelDrift(TrainTestCheck):
 
     def __init__(
             self,
-            max_num_categories: int = 10
+            alternative_label_transformers: List[Dict[str, Any]] = None
     ):
         super().__init__()
-        self.max_num_categories = max_num_categories
+        self.alternative_label_transformers = alternative_label_transformers
```

Comment:
Can we find an alternative to "transformers"? E.g. "properties" or "property_calculators"? Just that label_transformers may be confused with other stuff...

Reply:
Done